recipes/gmap/meta.yaml

```diff
@@ -14,6 +14,7 @@ source:
 
 build:
   number: 0
+  skip: True  # [osx and x86_64]
```

Comment:
@mencian I have the feeling you meant to add `- osx-arm64` to the additional platforms here

Reply:
@martin-g I tried building on `osx-arm64` but it's running into the same error as `osx x86_64`, so I opted to remove it from `additional-platforms` and skip `osx x86_64` specifically instead. I'll revisit this recipe later to build on `osx x86_64` and `osx-arm64`.